apis/cli/v1alpha1/catalog_types.go

```diff
@@ -102,6 +132,20 @@ type PluginDescriptor struct {
 	// Aliases are other text strings used to call this command
 	Aliases []string `json:"aliases,omitempty" yaml:"aliases,omitempty"`
 
+	// InstallationPath is a relative installation path for a plugin binary.
+	// E.g., cluster/v0.3.2@sha256:...
+	InstallationPath string `json:"installationPath"`
+
+	// Discovery is the name of the discovery from where
+	// this plugin is discovered.
+	Discovery string `json:"discovery"`
+
+	// Scope is the scope of the plugin. Stand-Alone or Context
+	Scope string `json:"scope"`
```

Comment:
Should we define `standalone` and `context` constants for scope here? Is this validated somewhere?

Reply:
Yeah. We should. But would like to defer it to later when we are actually using that so that we can confirm and do not add additional things if constants are not needed.